pipelines/synapse-release.yaml

```diff
@@ -303,12 +304,31 @@ stages:
 
             # Re-enable triggers in the target Synapse Workspace
             - task: AzureSynapseWorkspace.synapsecicd-deploy.toggle-trigger.toggle-triggers-dev@2
-              condition: succeededOrFailed()
-              displayName: 'Enable Synapse Triggers'
+              condition: and(succeededOrFailed(), eq('${{ parameters.environment }}', 'Dev'))
+              displayName: 'Enable Synapse Triggers - Dev'
               inputs:
                 azureSubscription: ${{ variables.armServiceConnectionName }}
                 ResourceGroupName: $(data_resource_group_name)
                 WorkspaceName: $(synapse_workspace_name)
+                Triggers: 'tr_daily_weekdays_1500,tr_backup_daily,tr_weekly'
+
+            - task: AzureSynapseWorkspace.synapsecicd-deploy.toggle-trigger.toggle-triggers-dev@2
+              condition: and(succeededOrFailed(), eq('${{ parameters.environment }}', 'Test'))
+              displayName: 'Enable Synapse Triggers - Test'
+              inputs:
+                azureSubscription: ${{ variables.armServiceConnectionName }}
+                ResourceGroupName: $(data_resource_group_name)
+                WorkspaceName: $(synapse_workspace_name)
+                Triggers: 'tr_daily_weekdays_1500,tr_backup_daily,tr_weekly'
+
+            - task: AzureSynapseWorkspace.synapsecicd-deploy.toggle-trigger.toggle-triggers-dev@2
+              condition: and(succeededOrFailed(), eq('${{ parameters.environment }}', 'Prod'))
+              displayName: 'Enable Synapse Triggers - Prod'
+              inputs:
+                azureSubscription: ${{ variables.armServiceConnectionName }}
+                ResourceGroupName: $(data_resource_group_name)
+                WorkspaceName: $(synapse_workspace_name)
+                Triggers: 'tr_daily_7days_2100,tr_backup_daily,tr_weekly'

```

Comment:
This is alot of repeated code. Perhaps we can add a variable and set it depending on env, then just have one task, e.g.

```
variables:
...
- name: synapseTriggers
  ${{ if eq(parameters.environment , 'Dev') }}:
    value: 'tr_daily_weekdays_1500,tr_backup_daily,tr_weekly'
  ${{ if eq(parameters.environment , 'Test') }}:
    value: 'tr_daily_weekdays_1500,tr_backup_daily,tr_weekly'
  ${{ if eq(parameters.environment , 'Prod') }}:
    value: 'tr_daily_7days_2100,tr_backup_daily,tr_weekly'

...

           - task: AzureSynapseWorkspace.synapsecicd-deploy.toggle-trigger.toggle-triggers-dev@2
              condition: succeededOrFailed()
              displayName: 'Enable Synapse Triggers'
              condition: succeededOrFailed()
              displayName: 'Enable Synapse Triggers - Dev'
              inputs:
                azureSubscription: ${{ variables.armServiceConnectionName }}
                ResourceGroupName: $(data_resource_group_name)
                WorkspaceName: $(synapse_workspace_name)
                Triggers: ${{ variables.synapseTriggers }}
```

Reply:
Good point. I've cleaned it up. Thanks.